### ISSUE TITLE
Could not open library '/usr/share/gems/gems/sassc-2.4.0/lib/sassc/libsass.so

### ISSUE THREAD

**Issue by marcosgopen** [wildfly/wildfly.org#424]
I encountered an issue serving the website on a docker container (fedora:latest)

Reproducer:

docker run -it -p 4000:4000  fedora 
    1  sudo dnf install ruby ruby-devel openssl-devel redhat-rpm-config @development-tools
    2  gem -v
    3  g++ -v
    4  yum install g++
    5  make -v
    6  gcc -v
    7   gem install bundler
    8  git clone https://github.com/wildfly/wildfly.org
    9   cd wildfly.org
   10   bundle install
   11   bundle exec jekyll serve

Error:
```
# bundle exec jekyll serve
Calling `DidYouMean::SPELL_CHECKERS.merge!(error_name => spell_checker)' has been deprecated. Please call `DidYouMean.correct_error(error_name, spell_checker)' instead.
bundler: failed to load command: jekyll (/usr/bin/jekyll)
/usr/share/gems/gems/ffi-1.15.5/lib/ffi/library.rb:145:in `block in ffi_lib': Could not open library '/usr/share/gems/gems/sassc-2.4.0/ext/libsass.so': /usr/share/gems/gems/sassc-2.4.0/ext/libsass.so: cannot open shared object file: No such file or directory (LoadError)
	from /usr/share/gems/gems/ffi-1.15.5/lib/ffi/library.rb:99:in `map'
	from /usr/share/gems/gems/ffi-1.15.5/lib/ffi/library.rb:99:in `ffi_lib'
	from /usr/share/gems/gems/sassc-2.4.0/lib/sassc/native.rb:13:in `rescue in <module:Native>'
	from /usr/share/gems/gems/sassc-2.4.0/lib/sassc/native.rb:10:in `<module:Native>'
	from /usr/share/gems/gems/sassc-2.4.0/lib/sassc/native.rb:6:in `<module:SassC>'
	from /usr/share/gems/gems/sassc-2.4.0/lib/sassc/native.rb:5:in `<top (required)>'
	from /usr/share/gems/gems/sassc-2.4.0/lib/sassc.rb:31:in `require_relative'
	from /usr/share/gems/gems/sassc-2.4.0/lib/sassc.rb:31:in `<top (required)>'
	from /usr/share/gems/gems/jekyll-sass-converter-2.2.0/lib/jekyll/converters/scss.rb:3:in `require'
	from /usr/share/gems/gems/jekyll-sass-converter-2.2.0/lib/jekyll/converters/scss.rb:3:in `<top (required)>'
	from /usr/share/gems/gems/jekyll-sass-converter-2.2.0/lib/jekyll-sass-converter.rb:4:in `require'
	from /usr/share/gems/gems/jekyll-sass-converter-2.2.0/lib/jekyll-sass-converter.rb:4:in `<top (required)>'
	from /usr/share/gems/gems/jekyll-4.2.2/lib/jekyll.rb:195:in `require'
	from /usr/share/gems/gems/jekyll-4.2.2/lib/jekyll.rb:195:in `<top (required)>'
	from /usr/share/gems/gems/jekyll-4.2.2/exe/jekyll:8:in `require'
	from /usr/share/gems/gems/jekyll-4.2.2/exe/jekyll:8:in `<top (required)>'
	from /usr/bin/jekyll:25:in `load'
	from /usr/bin/jekyll:25:in `<top (required)>'
	from /usr/share/gems/gems/bundler-2.2.19/lib/bundler/cli/exec.rb:63:in `load'
	from /usr/share/gems/gems/bundler-2.2.19/lib/bundler/cli/exec.rb:63:in `kernel_load'
	from /usr/share/gems/gems/bundler-2.2.19/lib/bundler/cli/exec.rb:28:in `run'
	from /usr/share/gems/gems/bundler-2.2.19/lib/bundler/cli.rb:474:in `exec'
	from /usr/share/gems/gems/bundler-2.2.19/lib/bundler/vendor/thor/lib/thor/command.rb:27:in `run'
	from /usr/share/gems/gems/bundler-2.2.19/lib/bundler/vendor/thor/lib/thor/invocation.rb:127:in `invoke_command'
	from /usr/share/gems/gems/bundler-2.2.19/lib/bundler/vendor/thor/lib/thor.rb:392:in `dispatch'
	from /usr/share/gems/gems/bundler-2.2.19/lib/bundler/cli.rb:30:in `dispatch'
	from /usr/share/gems/gems/bundler-2.2.19/lib/bundler/vendor/thor/lib/thor/base.rb:485:in `start'
	from /usr/share/gems/gems/bundler-2.2.19/lib/bundler/cli.rb:24:in `start'
	from /usr/share/gems/gems/bundler-2.2.19/exe/bundle:49:in `block in <top (required)>'
	from /usr/share/gems/gems/bundler-2.2.19/lib/bundler/friendly_errors.rb:130:in `with_friendly_errors'
	from /usr/share/gems/gems/bundler-2.2.19/exe/bundle:37:in `<top (required)>'
	from /usr/local/bin/bundle:25:in `load'
	from /usr/local/bin/bundle:25:in `<main>'
/usr/share/gems/gems/ffi-1.15.5/lib/ffi/library.rb:145:in `block in ffi_lib': Could not open library '/usr/share/gems/gems/sassc-2.4.0/lib/sassc/libsass.so': /usr/share/gems/gems/sassc-2.4.0/lib/sassc/libsass.so: cannot open shared object file: No such file or directory (LoadError)
	from /usr/share/gems/gems/ffi-1.15.5/lib/ffi/library.rb:99:in `map'
	from /usr/share/gems/gems/ffi-1.15.5/lib/ffi/library.rb:99:in `ffi_lib'
	from /usr/share/gems/gems/sassc-2.4.0/lib/sassc/native.rb:11:in `<module:Native>'
	from /usr/share/gems/gems/sassc-2.4.0/lib/sassc/native.rb:6:in `<module:SassC>'
	from /usr/share/gems/gems/sassc-2.4.0/lib/sassc/native.rb:5:in `<top (required)>'
	from /usr/share/gems/gems/sassc-2.4.0/lib/sassc.rb:31:in `require_relative'
	from /usr/share/gems/gems/sassc-2.4.0/lib/sassc.rb:31:in `<top (required)>'
	from /usr/share/gems/gems/jekyll-sass-converter-2.2.0/lib/jekyll/converters/scss.rb:3:in `require'
	from /usr/share/gems/gems/jekyll-sass-converter-2.2.0/lib/jekyll/converters/scss.rb:3:in `<top (required)>'
	from /usr/share/gems/gems/jekyll-sass-converter-2.2.0/lib/jekyll-sass-converter.rb:4:in `require'
	from /usr/share/gems/gems/jekyll-sass-converter-2.2.0/lib/jekyll-sass-converter.rb:4:in `<top (required)>'
	from /usr/share/gems/gems/jekyll-4.2.2/lib/jekyll.rb:195:in `require'
	from /usr/share/gems/gems/jekyll-4.2.2/lib/jekyll.rb:195:in `<top (required)>'
	from /usr/share/gems/gems/jekyll-4.2.2/exe/jekyll:8:in `require'
	from /usr/share/gems/gems/jekyll-4.2.2/exe/jekyll:8:in `<top (required)>'
	from /usr/bin/jekyll:25:in `load'
	from /usr/bin/jekyll:25:in `<top (required)>'
	from /usr/share/gems/gems/bundler-2.2.19/lib/bundler/cli/exec.rb:63:in `load'
	from /usr/share/gems/gems/bundler-2.2.19/lib/bundler/cli/exec.rb:63:in `kernel_load'
	from /usr/share/gems/gems/bundler-2.2.19/lib/bundler/cli/exec.rb:28:in `run'
	from /usr/share/gems/gems/bundler-2.2.19/lib/bundler/cli.rb:474:in `exec'
	from /usr/share/gems/gems/bundler-2.2.19/lib/bundler/vendor/thor/lib/thor/command.rb:27:in `run'
	from /usr/share/gems/gems/bundler-2.2.19/lib/bundler/vendor/thor/lib/thor/invocation.rb:127:in `invoke_command'
	from /usr/share/gems/gems/bundler-2.2.19/lib/bundler/vendor/thor/lib/thor.rb:392:in `dispatch'
	from /usr/share/gems/gems/bundler-2.2.19/lib/bundler/cli.rb:30:in `dispatch'
	from /usr/share/gems/gems/bundler-2.2.19/lib/bundler/vendor/thor/lib/thor/base.rb:485:in `start'
	from /usr/share/gems/gems/bundler-2.2.19/lib/bundler/cli.rb:24:in `start'
	from /usr/share/gems/gems/bundler-2.2.19/exe/bundle:49:in `block in <top (required)>'
	from /usr/share/gems/gems/bundler-2.2.19/lib/bundler/friendly_errors.rb:130:in `with_friendly_errors'
	from /usr/share/gems/gems/bundler-2.2.19/exe/bundle:37:in `<top (required)>'
	from /usr/local/bin/bundle:25:in `load'
	from /usr/local/bin/bundle:25:in `<main>
```
Am I missing something?

**Comment by marcosgopen** [comment]
This is a duplicate of https://github.com/sass/sassc-ruby/issues/146

**Comment by marcosgopen** [comment]
I solved it changing sassc to 2.1.0 in Gemfile.lock
